challenges/objects-arrays.js

```diff
@@ -46,7 +70,13 @@ const graduates = [{"id":1,"first_name":"Cynde","university":"Missouri Southern
 
 Once you have the new array created, sort the universities alphabetically and log the result. */
 const universities = [];
-console.log(universities)
+
+for(i = 0; i < graduates.length; i++) {
+  universities.push(graduates[i]["university"]);
+  universities.sort();
+}
```

Comment:
Careful, currently sorting every iteration. Move outside the loop and it will only sort once.